### PR TITLE
Updating menu options

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7517

### CODE REVIEW

File: Teams/set-up-communications-credits-for-your-organization.md [hunk 1]
```diff
@@ -55,11 +55,13 @@ For more information, see [Microsoft Teams add-on licensing](./teams-add-on-lice
 
 1. Sign in to the [Microsoft 365 admin center](https://portal.office.com/Adminportal) with your work or school account.
     
-2. In the left navigation of the Microsoft 365 admin center, go to **Billing** > **Purchase Services**. Scroll down and select **Add-Ons**.
+2. In the left navigation of the Microsoft 365 admin center, go to **Billing** > **Purchase Services**.
 
-3. Select **Communications Credits**.
+3. Look for **Communication Credits** under the **add-ons** category or search for "Communication Credits" in the **Search all product categories** search box and click **Details**.
```
Comment (JohanFreelancer9) [inline Teams/set-up-communications-credits-for-your-organization.md:60]:
```suggestion
3. Look for **Communication Credits** under the **add-ons** category or search for "Communication Credits" in the **Search all product categories** search box and select **Details**.
```

File: Teams/set-up-communications-credits-for-your-organization.md [hunk 2]
```diff
@@ -55,11 +55,13 @@ For more information, see [Microsoft Teams add-on licensing](./teams-add-on-lice
...
     
-4. On the **Communications Credits** subscription page, fill in your information, and then click **Next**:
+4. Review the service information and select **Buy**. (Note: A fixed number of Communication Credits licenses are automatically selected in every order).
```
Comment (JohanFreelancer9) [inline Teams/set-up-communications-credits-for-your-organization.md:62]:
```suggestion
4. Review the service information and select **Buy**. (Note: A fixed number of Communication Credits licenses are automatically selected in every order.)
```

File: Teams/set-up-communications-credits-for-your-organization.md [hunk 3]
```diff
@@ -76,7 +78,7 @@ For more information, see [Microsoft Teams add-on licensing](./teams-add-on-lice
      > 
      > When using the auto-recharge function, invoicing for Communication Credits is generated when the trigger amount is reached and a recharge transaction is processed. Communication credit amounts are used in a first in first out manner. To learn how to check your monthly usage, read [Microsoft Teams PSTN usage report](/microsoftteams/teams-analytics-and-reports/pstn-usage-report).
     
-5. Enter your payment information and click **Place order**.
+6. Click **Place order**.
```
Comment (JohanFreelancer9) [inline Teams/set-up-communications-credits-for-your-organization.md:81]:
```suggestion
6. Select **Place order**.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**